extensions/amp-story/0.1/amp-story.js

```diff
@@ -795,7 +822,20 @@ export class AmpStory extends AMP.BaseElement {
         this.updateBackground_(this.activePage_.element);
       }
     } else {
-      this.element.removeAttribute('desktop');
+      this.vsync_.run({
+        measure: state => {
+          const {offsetWidth, offsetHeight} = this.element;
+          state.isLandscape = offsetWidth > offsetHeight;
+        },
+        mutate: state => {
+          if (state.isLandscape) {
```

Comment:
Please change if/else clause to

`this.element.classList.toggle(LANDSCAPE_OVERLAY_CLASS, state.isLandscape)`

Reply:
Done